app/styles/components/_offering-editor.scss

```diff
@@ -0,0 +1,48 @@
+.offering-editor {
```

Comment:
This form looks great, but it should be styled with the existing form styles in place on the rest of the site and match those other forms.  Otherwise its too hard to maintain long term.